api/base/settings/defaults.py

```diff
@@ -104,6 +104,16 @@
         'api.base.authentication.drf.OSFSessionAuthentication',
         'api.base.authentication.drf.OSFCASAuthentication'
     ),
+    'DEFAULT_THROTTLE_CLASSES': (
+        'rest_framework.throttling.UserRateThrottle',
+        'api.base.throttling.NonCookieAuthThrottle',
+        'api.base.throttling.AddContributorThrottle',
```

Comment:
Should this be a default throttle class? I think this only gets applied to a small number of routes.